content/en/cosign/sign.md

```diff
@@ -161,19 +161,153 @@ The upload is skipped by using the `--upload=false` flag (default true). To capt
 $ cosign sign --key key.pem --upload=false --output-signature demo.sig --output-certificate demo.crt user/demo
 ```
 
-## Generate the signature payload (to sign with another tool)
+## Generate the Signature Payload With `cosign` (to sign with another tool)
 
-The json payload is printed to stdout:
+You can also use other tools for signing - not just `cosign`. This section will provide examples of how to sign with tools other than `cosign`.
+
+### `openssl`
```

Comment:
This is already covered under https://github.com/sigstore/docs/blob/54fe80cc9d6f19e23b2bc953efba60d148edeb28/content/en/cosign/sign.md?plain=1#L137-L142

Reply:
Removed :+1: